packages/drupal-kit/package.json

```diff
@@ -50,18 +50,18 @@
     "rollup": "rollup -c"
   },
   "devDependencies": {
-    "@apollo/client": "^3.5.10",
-    "@types/humps": "^2.0.1",
+    "@apollo/client": "^3.6.9",
+    "@types/humps": "^2.0.2",
     "@types/isomorphic-fetch": "^0.0.35",
-    "graphql": "^0.11.3 || ^0.12.3 || ^0.13.0 || ^14.0.0 || ^15.0.0",
+    "graphql": "^15.8.0",
     "humps": "^2.0.1",
     "isomorphic-fetch": "^3.0.0",
-    "jsona": "^1.9.7",
-    "react": ">=16.8.0 <17.0.0",
-    "react-dom": "^16.8.0",
-    "typescript": ">=3.8 <5.0"
+    "jsona": "^1.10.1",
+    "react": "^16.14.0",
+    "react-dom": "^16.14.0",
+    "typescript": "^4.7.4"
```

Comment:
We don't want to introduce potentially breaking changes by shotgun-upgrading all dependencies.